tests/testthat/test-deprecated.R

```diff
@@ -35,6 +35,20 @@ test_that("error coverts _soft and _warn to errors", {
   expect_defunct(deprecate_stop("1.0.0", foo()))
 })
 
+test_that("soft deprecation not shown if you can't do anything about it", {
+  local_options(lifecycle_verbosity = "default")
+
```

Comment:
Needs a `local_bindings(.env = global_env())` to define `softly`?

Reply:
`softly()` has to be defined inside a package environment so that `env_inherits_global()` is `FALSE`

Reply:
hmm only the calling user env is checked. I don't think binding softly or softly-softly in the global env would change the relevant state so it should work. softly-softly will still inherit from a package env.

Equivalently you can use `lifecycle:::`, IIRC this is allowed by R CMD check?